doc/source/index.rst

```diff
@@ -328,3 +327,14 @@ jupyter_execute_kwargs
 
     Keyword arguments to pass to ``nbconvert.preprocessors.execute.executenb``, which controls how
     code cells are executed. The default is ``dict(timeout=-1, allow_errors=True)``.
+
+jupyter_sphinx_linenos
+
+    Boolean argument it use line numbering in code cell display by default. Equivalent
+    to setting the `linenos` directive for all code cells.
```

Comment:
It might not be obvious to the user what we call "code cells", I think we should avoid calling the directives "code cells" unless we clearly introduce the concept.

Reply:
Ok, thanks, that is much clearer. I have adjusted the docs. Please check those out. I second going ahead with this version.